website/content/v1.7/talos-guides/install/cloud-platforms/akamai.md

```diff
@@ -151,3 +151,9 @@ We can also watch the cluster bootstrap via:
 ```bash
 talosctl --talosconfig talosconfig health
 ```
+
+We can also watch the node overview, logs and real-time metrics dashboard via:
+
+```bash
+talosctl  --talosconfig talosconfig dashboard
```

Comment:
```suggestion
talosctl --talosconfig talosconfig dashboard
```

Reply:
@MindTooth I removed space according to your suggestion and also fixed AWS docs from where I copied the line 😉